src/Compilers/Server/VBCSCompiler/NamedPipeClientConnection.cs

```diff
@@ -11,6 +11,7 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CommandLine;
 using System.IO.Pipes;
+using static Microsoft.CodeAnalysis.CommandLine.CompilerServerLogger;
```

Comment:
Is this needed? Hard to see from the review if we're still using Log/LogException elsewhere other than the ones you removed. The only place you added explicitly calls into CompilerServerLogger

Reply:
It was used on line 131 but I decided to delete this and use `CompilerServerLogger` there as it's the predominate style in that file.